### PR TITLE
Bug fixes for `evaluate` function in `cli.py`

### PR DESCRIPTION
**Pull request by pk1130**
**Reference Issues/PRs**

Contributes to issue #19.

**What does this implement/fix? Explain your changes.**

Fixed bug in Line 144 of `cli.py`. 
Completed debugging and PR as requested in conversation of the same issue. 
Comments above respective lines explain the fixes.

**Any other comments? Anything else to-do?**

Need to create test suite of unit/integration tests. @EntilZha @jplalor please take a look when you have some time. Will add unit/integration test when time permits. For now, the issue has been fixed. Comments and commit changes should show the error and what has been changed. Please let me know in case of any issues/suggestions/questions. Thanks!

### CODE REVIEW

File: py_irt/cli.py [hunk 1]
```diff
@@ -122,7 +122,8 @@ def evaluate(
     seed: int = 42,
     train_size: float = 0.9,
 ):
-    console.log(f"model_type: {model_type} data_path: {data_path}")
+    # removed console.log since data_path is not passed into this function as an
```
Comment (EntilZha) [inline py_irt/cli.py:125]: This comment doesn't need to be here and the model type should still be logged.

Comment (pk1130) [reply py_irt/cli.py]: Resolved in next commit. Does everything else look good?

File: py_irt/cli.py [hunk 2]
```diff
@@ -141,7 +141,9 @@ def evaluate(
         )
 
     observation_subjects = [entry["subject_id"] for entry in subject_item_pairs]
-    observation_items = [entry["item_id"] for entry in subject_item_pairs]
+    # since all subjects use same set of items for predictions (as visible from 
+    # squad.jsonlines), getting all items from one subject
+    observation_items = subject_item_pairs[0]["responses"].keys()
```
Comment (jplalor) [inline py_irt/cli.py:146]: See my comment in issue #19. Your change requires predictions for all items for a given subject, whereas the prior implementation is a bit more general. For your case, you'd just want to convert your subject-responses jsonlines file to a subject-item jsonlines file.

Comment (jplalor) [reply py_irt/cli.py:146]: FYI I've updated the README to specify the data format for evaluation to help with this.

**Comment by pk1130** [comment]
Apologies for the delay in implementing these changes. Currently a little busy with another research task. Will get to this later this week if that's ok? I read all the comments about the changes and agree with you @jplalor. Thanks for the detailed explanations. Now `test_pairs_path` and the `evaluate` function in `cli.py` make much more sense. I was updating the docs as you guys were explaining stuff, but looks like you already went ahead and updated the docs for evaluation as well. Thanks for that too @jplalor!

**Comment by jplalor** [comment]
No problem! I believe we can close this without merging unless I'm missing something. Let me know!

**Pull request closed without merging**